rosys/vision/mjpeg_camera/mjpeg_camera_provider.py

```diff
@@ -37,14 +37,17 @@ def __init__(self, *,
 
     def restore(self, data: dict[str, dict]) -> None:
         for camera_data in data.get('cameras', {}).values():
+            self.log.debug('restoring camera: %s', camera_data)
             camera = MjpegCamera.from_dict(camera_data)
             self.add_camera(camera)
         for camera in self._cameras.values():
             camera.NEW_IMAGE.register(self.NEW_IMAGE.emit)
 
     async def scan_for_cameras(self) -> list[tuple[str, str]]:
+        self.log.debug('scanning for cameras...')
         ids_ips: list[tuple[str, str]] = []
         async for mac, ip in find_cameras(self.network_interface):
+            self.log.debug('device found: %s %s', mac, ip)
```

Comment:
@NiklasNeugebauer this is taking quite a lot of space in the logs (and happens quite frequently). Maybe it's better to improve this. I think we should not force every project which activates DEBUG for RoSys to fine-granularly deactivate it for the mjpeg camera provider.